blogging/admin.py

```diff
@@ -0,0 +1,42 @@
+from django.contrib import admin
+from django.db import models
+from django.contrib.auth.models import User
+from blogging.models import Post, Category
+
+
+class PostInline(admin.TabularInline):
+    model = Category.posts.through
+
+
+class PostAdmin(admin.ModelAdmin):
+    inlines = [PostInline,
+               ]
+
+
+#  exclude  the ‘posts’ field from the form in your  Category  admin
+class CategoryInline(admin.TabularInline):
+    model = Category
+
```

Comment:
Not needed, but it looks like you were exploring different paths, and no big harm in creating classes you don't use.